spec/models/list_spec.rb

```diff
@@ -71,6 +71,40 @@
       expect(list.next_user).to eq(users(:aldhsu))
     end
 
+    it "is the second user when we exclude the first user on a new list" do
+      expect(list.tasks).to be_empty
+      expect(list.next_user).to eq(users(:aldhsu))
+      expect(list.next_user("aldhsu")).to eq(users(:alex))
+    end
+
+    it "is not the excluded user even when their turn is due" do
+      create_tasks(:aldhsu)
+      expect(list.tasks.size).to eq(1)
+      expect(list.next_user).to eq(users(:alex))
+      expect(list.next_user("alex")).to eq(users(:dave))
+    end
+
+    it "is not the excluded user even when their turn is due, covering list cycle" do
+      create_tasks(:aldhsu, :alex, :dave)
+      expect(list.tasks.size).to eq(3)
+      expect(list.next_user).to eq(users(:tate))
+      expect(list.next_user("tatey")).to eq(users(:aldhsu))
+    end
+
+    it "is not the excluded user even when their turn is due, covering list cycle" do
+      create_tasks(:aldhsu, :alex, :dave, :tate)
+      expect(list.tasks.size).to eq(4)
+      expect(list.next_user).to eq(users(:aldhsu))
+      expect(list.next_user("aldhsu")).to eq(users(:alex))
+    end
+
+    it "is not affected by an irrelevant exclusion" do
+      create_tasks(:aldhsu)
+      expect(list.tasks.size).to eq(1)
+      expect(list.next_user).to eq(users(:alex))
+      expect(list.next_user("aldhsu")).to eq(users(:alex))
+    end
+
```

Comment:
I think there are probably two more tests we should add to this suite:

1. what happens with an empty user list
1. what happens with a user list of one

Even if we don't like the results at present, we need to know how they handle it.